tools/value-generation/gen.py

```diff
@@ -134,6 +172,9 @@ def processCluster(crd_value, value_map, comment_map) :
   # Set this empty to ensure we auto-generate it by default
   value_map[crd_value]['security']['adminSecret'] = ''
 
+  # Set monitoring to emtpy since it requires image which is not provided by default
```

Comment:
And for 7.0.2 we can use the server image to provide it.

Reply:
Indeed. for 2.2.3 this will be the move